src/components/Navbar/Navbar.tsx

```diff
@@ -68,36 +47,34 @@ const Navbar = (props: Props) => {
       </NavDropdown>
     )
   }
-
   return (
     <NavbarRB bg={bg} variant={variant}>
-      <NavbarRB.Brand href={brand.href} onClick={brand.onClick}>
+      <NavbarRB.Brand href={brand.href} onClick={brand.onClick} style={{ cursor: 'pointer' }}>
```

Comment:
thanks for making this change 😊. It was bugging me, and I forgot to mention it in the issue.